.gitignore

```diff
@@ -1,3 +1,5 @@
+.env
```

Comment:
Shit I think I forgot to add it back when I took it out, after deploying to heroku sorry.